### ISSUE TITLE
Pairwise Logistic Loss (paper typo)

### ISSUE THREAD

**Issue by eggie5** [tensorflow/ranking#23]
In the TF Ranking paper, equation 4, for the Pairwise Logistic Loss shows:

```
log(1 + exp(pairwise_logits) 
```

but, I think it's missing a negative sign. Should be:

```
log(1 + exp(-pairwise_logits)
```

Otherwise a correct ranking would incur a large loss penalty.

**Comment by bendersky** [comment]
Alex,
You are correct -- this is a typo. it will be fixed in the next version of the paper that we're currently working on.
